RELEASES.md

```diff
@@ -1,3 +1,80 @@
+Version 1.42.0 (2020-03-12)
+==========================
+
+Language
+--------
+- [You can now use `#[repr(transparent)]` on univariant `enum`s.][68122] Meaning
+  that you can create an enum that has the exact layout and ABI of the type
+  it contains.
+- [You can now use the slice pattern syntax with subslices.][67712] e.g.
+  ```rust
+  fn foo(words: &[&str]) {
+      match words {
+          ["Hello", "World", "!", ..] => println!("Hello World!"),
+          ["Foo", "Bar", ..] => println!("Baz"),
+          rest => println!("{}", rest),
+      }
+  }
+  ```
+  <!-- TODO -->
+- [Merge `TraitItem` & `ImplItem into `AssocItem`][67131]
+
+Compiler
+--------
+- [Added tier 3\* support for `armv7a-none-eabi`.][68253]
+- [Added tier 3 support for `riscv64gc-unknown-linux-gnu`.][68037]
```

Comment:
This was also made a tier 2 target in https://github.com/rust-lang/rust/pull/68339